src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,104 @@
-""" This is a dummy example """
+import requests
+import json
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from datetime import timedelta
+from ratelimit import limits, sleep_and_retry
+
+
+COORDINATES = {
+    "Madrid": {"latitude": 40.416775, "longitude": -3.703790},
+    "London": {"latitude": 51.507351, "longitude": -0.127758},
+    "Rio": {"latitude": -22.906847, "longitude": -43.172896},
+}
+
+VARIABLES = "temperature_2m_mean,precipitation_sum,soil_moisture_0_to_10cm_mean"
+
+
+def get_data_meteo_api(lat: float, long: float, variable: str):
+    # For the API call I only parametrized the coordinates as
```

Comment:
and yet, this string is too long and hard to read, specially because we may want to parametrize it later. So parametrize things like the `start_date`, `end_date`...